### PR TITLE
Getting the code up to date with what is in production

### PR DESCRIPTION
**Pull request by mrpatrickwest**
The code in git is different from production in that paths have been
changed. Makes it a little more difficult to update the github
repository without changing the modifications made for the github
directory structure.

### CODE REVIEW

File: people.html [hunk 1]
```diff
@@ -35,10 +35,10 @@
 					'paradigm' : 'http://escience.rpi.edu/ontology/sesf/s2s-core/4/0/HierarchicalSearch',
 					'inputs' : [
 						{'name' : s2sConfOntURI + 'SearchNames', 'enabled' : true, 'widget' : s2sConfOntURI + 'GenericWidget'},
-						{'name' : 'http://a9.com/-/opensearch/extensions/geo/1.0/box', 'enabled' : true, 'widget' : s2sConfOntURI + 'WmsOpenLayersBoundingBox'},
-						{'name' : s2sConfOntURI + 'Organizations', 'enabled' : true, 'widget' : s2sConfOntURI + 'ContextSelect'},
-						{'name' : s2sConfOntURI + 'Communities', 'enabled' : true, 'widget' : s2sConfOntURI + 'ContextSelect'},
-						{'name' : s2sConfOntURI + 'Groups', 'enabled' : true, 'widget' : s2sConfOntURI + 'ContextSelect'},
+						{'name' : s2sConfOntURI + 'box', 'enabled' : false, 'widget' : s2sConfOntURI + 'WmsOpenLayersBoundingBox'},
```
Comment (zednis) [inline people.html:38]: Just to confirm - we want the Organization, Community, and Group facets to be disabled by default?

**Comment by zednis** [comment]
I had one comment about widgets being disabled in the people browser.  Everything else looks good.

File: people.html (hunk 1 above)
Comment (mrpatrickwest) [inline people.html:38]: Yes, by default they are disabled. Wanted to make it look like the Drupal DIRECTORY as much as posslbe. So the facets go left to right along the top and the result set underneath the facets.

Comment (zednis) [inline people.html:38]: OK, sound good.  I just wanted to make sure.

**Comment by zednis** [comment]
Everything looks good.  :shipit:

**Pull request merged**